### PR TITLE
Samsung.EasySettingBox version 1.2.1

### PR DESCRIPTION
**Pull request by upintheairsheep**
- [ ] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [ ] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [ ] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`?
- [ ] Have you tested your manifest locally with `winget install --manifest <path>`?
- [ ] Does your manifest conform to the [1.4 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.4.0)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

 ###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/99335)

### CODE REVIEW

File: manifests/s/Samsung/EasySettingBox/1.2.1/Samsung.EasySettingBox.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,14 @@
+# Created using wingetcreate 1.2.5.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.4.0.schema.json
+
+PackageIdentifier: Samsung.EasySettingBox
+PackageVersion: 1.2.1
+Installers:
+- InstallerUrl: https://org.downloadcenter.samsung.com/downloadfile/ContentsFile.aspx?CDSite=US&CttFileID=8609826&CDCttType=SW&ModelType=C&ModelName=LS27B804TGNXGO&VPath=SW/202301/20230106083833260/EasySettingMSIX64.msi
```
Comment (Trenly) [inline manifests/s/Samsung/EasySettingBox/1.2.1/Samsung.EasySettingBox.installer.yaml:7]:
```suggestion
- InstallerUrl: https://downloadcenter.samsung.com/content/SW/202301/20230106083833260/EasySettingMSIX64.msi
```

**Review by Trenly — CHANGES_REQUESTED** [review]
```
CustomAction _183B2A37_C02E_4615_A5F5_A511B36654E5 returned actual error code 1157 (note this may not be 100% accurate if translation happened inside sandbox)
MSI (s) (F4:70) [10:27:25:453]: Product: Easy Setting Box -- Error 1723. There is a problem with this Windows Installer package. A DLL required for this install to complete could not be run. Contact your support personnel or package vendor.  Action _183B2A37_C02E_4615_A5F5_A511B36654E5, entry: _EsbInstall@4, library: C:\Program Files\Samsung\Easy Setting Box\InstallerHelper.dll 

Error 1723. There is a problem with this Windows Installer package. A DLL required for this install to complete could not be run. Contact your support personnel or package vendor.  Action _183B2A37_C02E_4615_A5F5_A511B36654E5, entry: _EsbInstall@4, library: C:\Program Files\Samsung\Easy Setting Box\InstallerHelper.dll 
Action ended 10:27:25: InstallExecute. Return value 3.
```

It also looks like this may have a dependency on the .NET Framework, and IIS 5.1
image

**Comment by ItzLevvie** [comment]
Close with reason: Stale pull request;

**Comment by upintheairsheep** [comment]
Re-open please

**Pull request closed without merging**